src/rust/ide/view/graph-editor/src/component/breadcrumbs/project_name.rs

```diff
@@ -30,8 +30,9 @@ use logger::enabled::Logger;
 // === Constants ===
 // =================
 
-/// Project name used as a placeholder in `ProjectName` view when it's initialized.
-pub const UNKNOWN_PROJECT_NAME : &str = "Unknown";
+// Project name used as a placeholder in `ProjectName` view when it's initialized.
+// This should never be visible, but if it is we want to easily trace it back here.
```

Comment:
"This should never be visible, but if it is we want to easily trace it back here" - I completely do not understand this sentence. Moreover, I do not understand the reason for this constant, but maybe it would be obvious after the sentence would be fixed :)

Reply:
This is a default value for the project name. But the project name should always be initialized externally for the current project. If this value is visible, it was not set to the correct project name due to some bug. If we leave this value, e.g., as empty string, we will have a harder time tracking the behaviour "no project name visible" in case something goes wrong. This is what happened here, we saw project name "Unknown", which was the previous default value, but this was not a clear indicator of the actual bug: faulty initialization.

Reply:
Ok, makes sense. Could you expand the comment there slightly to explain it better, a you just did to me here? <3